src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -434,4 +434,39 @@
     <value>{0} {1}
     The default value '{1}' is not a valid value for {0}.</value>
   </data>
+  <data name="AddRefPostActionFailed" xml:space="preserve">
+    <value>Failed to add reference {0} to project file {1}</value>
+  </data>
+  <data name="AddRefPostActionNoRefConfigured" xml:space="preserve">
+    <value>No references configured to be added to a project.</value>
```

Comment:
Shouldn't this be "No references configured to be added to project." so removing the 'a'. I'll leave this to @seancpeters because he is better with words than I am.